data/darktableconfig.xml.in

```diff
@@ -1532,6 +1532,20 @@
     <shortdescription>demosaicing for zoomed out darkroom mode</shortdescription>
     <longdescription>interpolation when not viewing 1:1 in darkroom mode: bilinear is fastest, but not as sharp. middle ground is using PPG + interpolation modes specified below, full will use exactly the settings for full-size export. X-Trans sensors use VNG rather than PPG as middle ground.</longdescription>
   </dtconfig>
+  <dtconfig prefs="darkroom" >
+    <name>preview_downsampling</name>
+    <type>
+      <enum>
+        <option>1.0 (none)</option>
```

Comment:
Since we are anyway comparing the values to set dev->preview_downsampling maybe we should use more user friendly entries:

```
   none
   1 / 2
   1 / 3
   1 / 4
```

Or
```
   none
   half
   one third
   one fourth
```

?

Reply:
I went for 
```
original
to 1/2
to 1/3
to 1/4
```
Is ok?

Reply:
Ok to me.